api/api/test/test_authentication.py

```diff
@@ -72,44 +72,45 @@ def test_check_user(mock_raise_if_exc, mock_submit, mock_distribute_function, mo
     mock_raise_if_exc.assert_called_once()
 
 
-@patch('api.authentication.token_urlsafe', return_value='test_token')
+@patch('api.authentication.change_keyPair', return_value=["b'-----BEGIN PRIVATE KEY-----\n'", "b'-----BEGIN PUBLIC KEY-----\n'"])
```

Comment:
Not a real use case. `change_keyPair` will return bytes and no strings. Review this test as it will change.